services/movie.py

```diff
@@ -28,13 +33,14 @@ def create_movie(
     genres_ids: list = None,
     actors_ids: list = None,
```

Comment:
For consistency and clarity, consider using `list[int]` as the type annotation for `genres_ids` and `actors_ids` instead of `list`.